code/modules/mob/living/carbon/human/species/necromorph/divider/divider.dm

```diff
@@ -337,7 +337,3 @@ Reanimate can be used to take control of any already-headless corpse on the grou
 
 	offsets_left = list(S_NORTH = new /vector2(-56, -10), S_SOUTH = new /vector2(-24, -10), S_EAST = new /vector2(-36, -14), S_WEST = new /vector2(-36, -14))
 	offsets_right = list(S_NORTH = new /vector2(-28, -10), S_SOUTH = new /vector2(-48, -10), S_EAST = new /vector2(-40, -14), S_WEST = new /vector2(-36,-14))
-
-
-#undef LEFT_ARM_OFFSETS
```

Comment:
these are undefined for a reason, this change will break divider and tripod

Reply:
this definition does not exist.